doc/sources/memory-requirements.rst

```diff
@@ -0,0 +1,35 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _memory_requirements:
+
+###################
+Memory requirements 
+###################
+
+By default |intelex| algorithms run in multithread mode using all available threads. 
+Depending on this the amount of RAM consumed by optimized algorithms may be more than the stock Scikit-learn.
+
+SVM RAM consumption
+-------------------
+In single-thread mode both Scikit-learn and |intelex| consumes approximately the same amount of RAM.
+In default |intelex| multithread mode with ``N`` number of threads algorithm will consume in ``N`` times more RAM.
```

Comment:
```suggestion
In single-thread mode, both scikit-learn and |intelex| consume approximately the same amount of RAM.
In default multithread mode in |intelex|, an algorithm with ``N`` threads will consume ``N`` times more RAM.
```

is the second sentence about svm or does it apply to all algorithms? I think it should be placed elsewhere if it applies to all algorithms

also: maybe we need to highlight differences between single-thread vs. multi-thread modes somehow: for example, a table

Reply:
Maybe it's better to add table with one row and 3 columns (Algorithm, single-thread and multi-thread) and add there information about SVM? This may be more readable than a separate topic about SVM

Reply:
yes, I agree